### PR TITLE
Log JoyTeleopException

### PR DESCRIPTION
**Pull request by youtalk**
Current `joy_teleop.py` implementation passes JoyTeleopException without any logging.
It might be difficult to debug `config.yaml` description.

This PR improves it.

### CODE REVIEW

**Comment by bmagyar** [comment]
@artivis happy to chime in?

File: joy_teleop/joy_teleop/joy_teleop.py [hunk 1]
```diff
@@ -317,21 +317,20 @@ def update_actions(self, evt=None):
...
-        rclpy.shutdown()
-    except JoyTeleopException:
-        pass
+    except JoyTeleopException as e:
+        node.get_logger().error(e.message)
     except KeyboardInterrupt:
         pass
 
+    node.destroy_node()
+    rclpy.shutdown()
```
Comment (artivis) [inline joy_teleop/joy_teleop/joy_teleop.py]: Genuine question, aren't those two implicitly called? 
```python
node.destroy_node()
rclpy.shutdown()
```

**Comment by artivis** [comment]
Left a small question, otherwise LGTM.

File: joy_teleop/joy_teleop/joy_teleop.py (hunk 1 above)
Comment (bmagyar) [reply joy_teleop/joy_teleop/joy_teleop.py]: I'm not sure this is relevant enough to hold up this PR. Those lines were they part of the original code as well.

Comment (artivis) [reply joy_teleop/joy_teleop/joy_teleop.py]: It is not, that's why I started the sentence with 'Genuine question' ;)

**Pull request merged**